static/app/views/monitors/components/monitorCheckIns.tsx

```diff
@@ -136,7 +136,11 @@ function MonitorCheckIns({monitor, monitorEnvs, orgSlug}: Props) {
                   emptyCell
                 )}
                 {defined(checkIn.duration) ? (
-                  <Duration seconds={checkIn.duration / 1000} />
+                  <div>
+                    <Tooltip title={<Duration exact seconds={checkIn.duration / 1000} />}>
```

Comment:
Duration doesn’t have a built in tooltip option does it

Reply:
nope, no tooltip duration, doesn't seem like this pattern is used much elsewhere tho to be fair. also 👀 get back to sabbatical sir 😠!